### PR TITLE
Add graph

### PR DESCRIPTION
**Pull request by wbenica**

### CODE REVIEW

**Comment by wbenica** [comment]
I have bar, line and scatter working pretty well. just merging to see how it works with stats.

**Comment by ThomKaar** [comment]
rip, i've been doing the graph all day today as well

**Comment by ThomKaar** [comment]
your work looked good as well, if it is almost finished or finished we can use it.
Plus mine doesn't use the dataframes so we would have to change it quite a bit fix it up.

File: covidTracker.py [hunk 1]
```diff
@@ -31,26 +32,31 @@ def main():
 
     result = collection.aggregate(pipeline).next()
 
+    with open("./out/track_covid_usa", 'w') as f:
```
Comment (ThomKaar) [inline covidTracker.py:35]: Is this being used? or were you using it to debug code?

**Pull request closed without merging**